appVersion.sh

```diff
@@ -0,0 +1,3 @@
+# This script runs appVersion.js which fetches the version number from the
+# package.json file and writes it to a env variable.
+export REACT_APP_VERSION=`node appVersion.js`
```

Comment:
Yess a shell script that calls a node js script. Easiest way to extract a variable from the package.json and have it live as an env variable. Open to suggestions.

Reply:
I wonder if we can get rid of `appVersion.js` and do everything in this script. Could something like this work?

<img width="646" alt="Screen Shot 2022-01-28 at 5 21 10 PM" src="https://user-images.githubusercontent.com/47064971/151629502-1c44164f-122d-4d4f-9704-d4ff3bb20669.png">

(credits to this [stackoverflow post](https://stackoverflow.com/questions/47234740/read-name-property-of-package-json-file-with-bash) lol)

Reply:
Wow Great find! 🔥